src/gui/sdlmain.cpp

```diff
@@ -1740,25 +1804,22 @@ static SDL_Window * SetDefaultWindowMode()
 
 	if (sdl.desktop.fullscreen) {
 		sdl.desktop.lazy_init_window_size = true;
-		return GFX_SetSDLWindowMode(sdl.desktop.full.width,
-		                            sdl.desktop.full.height,
-		                            true,
-		                            sdl.desktop.want_type);
+		return SetWindowMode(sdl.desktop.want_type, sdl.desktop.full.width,
+		                     sdl.desktop.full.height, true,
+		                     sdl.desktop.want_resizable_window);
 	}
 
 	if (sdl.desktop.window.use_original_size) {
 		sdl.desktop.lazy_init_window_size = false;
-		return GFX_SetSDLWindowMode(sdl.draw.width,
-		                            sdl.draw.height,
-		                            false,
-		                            sdl.desktop.want_type);
+		return SetWindowMode(sdl.desktop.want_type, sdl.draw.width,
+		                     sdl.draw.height, false,
+		                     sdl.desktop.want_resizable_window);
```

Comment:
Similar to the above, a named bool for whatever the `true` and `false` is standing for in these calls to `SetWindowMode(...)` would make it immediately understandable.